### DISCUSSION TITLE
Question regarding the influence of material properties applied to a geometry on stress results

### DISCUSSION THREAD

**Discussion by clar2325** [robbievanleeuwen/section-properties#483] [Q&A]
Hi there,

I have been testing this package to study St. Venant's torsion and flexural shear stresses of arbitrary cross sections. I have a question regarding the application of material properties to a geometry with regards to the resultant shear stress. When I apply a material to the geometry, the results account for the shear modulus and are shifted. However, when I manipulate the elastic modulus, the results stay the same. It is only when I manipulate Poisson's ratio that the results are again shifted. My understanding is that any change to the shear modulus will influence the results of the shear stress distribution.

Why is it that results are only changed when Poisson's ratio is changed, and not also for the elastic modulus?

**Comment by robbievanleeuwen** [comment]
Hi @clar2325, I note that you closed this discussion post - just checking in to see if you figured out the answer to your question?

**Comment by clar2325** [comment]
Hi robbievanleeuwen,Yea I think I figured out that issue. Im very interested in your tool because I have also been trying to develop my own tool that performs a St Venant and flexural FEA analysis for an arbitrary cross section. I’ve been attempting this in MATLAB 2024b which has its own STL file import and meshing function. I’ve actually created a script that can calculate geometric properties and bending + axial stress distribution for an arbitrary cross section, but have been stuck on the combined torsion and flexural stress distribution. I saw a tool online called “Shape Builder” that references Gruttmann 1998’s paper on FEA for arbitrary cross sections, but I have struggled to implement the theory into MATLAB. One issue I have been running into recently while testing out your tool is DXF imports. I use both Solidworks and Siemens NX. It seems to work fine with linear geometry, I sketch on the top plane and apply a bounded surface (or planar) and then export as .dxf. However when I try some curved geometries I am running into some issues. Is this something you’ve run into? I might be exporting incorrectly and am curious how you’ve performed this procedure.Sincerely, Zachary ClarkOn Nov 9, 2024, at 6:37 PM, Robbie van Leeuwen ***@***.***> wrote:﻿
Hi @clar2325, I note that you closed this discussion post - just checking in to see if you figured out the answer to your question?

**Reply by robbievanleeuwen** [reply]
@clar2325 I recommend Analysis and Design of Elastic Beams: Computational Methods by Pilkey, see more in the [theory section of the docs](https://sectionproperties.readthedocs.io/en/stable/user_guide/theory.html).

The triangular mesh created by `sectionproperties` only has straight edges, so importing curved geometry would just discretise these curves into a number of straight lines. I don't import from dxf very often so don't have a whole lot of tips for you. I'm pretty sure `cad-to-shapely` (which `sectionproperties` uses) can handle splines, circles and arcs so maybe make sure you're exporting one of these? Otherwise you could discretise your curves to straight lines in your CAD program prior to exporting?

Best of luck!
